### PR TITLE
Fix mobilemeasurelength sketchstyle 2.2

### PR DESCRIPTION
**Pull request by pfirpfel**
Fixes: #3436

### CODE REVIEW

**Comment by pfirpfel** [comment]
I also added an example to the mobile_alt application to test the functionality.

File: contribs/gmf/apps/mobile_alt/js/controller.js [hunk 1]
```diff
@@ -58,6 +58,32 @@ app.AlternativeMobileController = function($scope, $injector) {
    */
   this.searchCoordinatesProjections = ['EPSG:21781', 'EPSG:2056', 'EPSG:4326'];
 
+
+  /**
+   * @type {ol.style.Style}
+   * @export
+   */
+  this.customMeasureStyle = new ol.style.Style({
```
Comment (fredj) [inline contribs/gmf/apps/mobile_alt/js/controller.js]: please add the corresponding `goog.require`

File: contribs/gmf/src/directives/mobilemeasurelength.js [hunk 2]
```diff
@@ -118,65 +136,85 @@ gmf.MobileMeasureLengthController = function($scope, ngeoDecorateInteraction, $f
...
+   * @type {boolean}
+   * @export
+   */
+  this.valid = false;
+};
+
+/**
+ * Initialise the controller.
+ */
+gmf.MobileMeasureLengthController.prototype.init = function() {
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/mobilemeasurelength.js]: Why don't we use the $onInit function from AngularJS ?

Comment (pfirpfel) [reply contribs/gmf/src/directives/mobilemeasurelength.js]: I think $onInit is exclusive for components, right?

I will refactor it into a component for 2.3

File: contribs/gmf/apps/mobile_alt/js/controller.js (hunk 1 above)
Comment (pfirpfel) [reply contribs/gmf/apps/mobile_alt/js/controller.js]: Added it.

File: contribs/gmf/src/directives/mobilemeasurelength.js (hunk 2 above)
Comment (ger-benjamin) [reply contribs/gmf/src/directives/mobilemeasurelength.js]: Oups, you're right

**Comment by pfirpfel** [comment]
The example works for point and length. I tested mobile_alt, which overrides the style, and mobile, which does not change it.

**Comment by sbrunner** [comment]
What's the status of this pull request?

**Comment by pfirpfel** [comment]
The example works, should I merge?

**Comment by sbrunner** [comment]
+1 :-)

**Pull request merged**